include/posix/time.h

```diff
@@ -10,6 +10,10 @@
 extern "C" {
 #endif
 
+#include <sys/timespec.h>
+#include <sys/_timeval.h>
```

Comment:
This doesn't work.  We don't provide those headers as part of minimal libc (newlib has them, though), so it's never going to be correct to pull them from somewhere else.

Reply:
libc is unalienable part of POSIX. There can't be POSIX without (no-nonsense) libc. Hence, there never will be POSIX with "minimal lib". OTOH, we have a pretty real libc, newlib. It's gotta become a requirement for POSIX subsys, and we're going to play along with it, not against it (or it'll keep biting us at the tail, like it has been doing last half year (last half year for me, since I started to look into POSIX subsys)). That's my current vision, we'll see how it goes.